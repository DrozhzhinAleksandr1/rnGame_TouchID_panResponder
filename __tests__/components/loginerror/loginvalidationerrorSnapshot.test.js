import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import {ErrorValidation}  from '../../../app/components/loginerror/loginvalidationerror';

it('create snapshot ErrorValidation', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<ErrorValidation />);
    expect(toolbarSnapshot).toMatchSnapshot();
});
