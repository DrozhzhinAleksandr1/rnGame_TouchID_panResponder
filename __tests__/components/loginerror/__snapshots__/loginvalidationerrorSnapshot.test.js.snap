// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot ErrorValidation 1`] = `null`;
